src/components/Todo.js

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import { HiPlusCircle } from 'react-icons/hi';
+import TodoList from './TodoList';
+
+const Todo = () => {
+  const [data, setDate] = useState([]);
```

Comment:
The setter function for the data state doesn't follow convention.
```suggestion
  const [data, setData] = useState([]);
```